src/helpers/SendEmail.ts

```diff
@@ -0,0 +1,29 @@
+import nodemailer from "nodemailer";
+
+const transporter = nodemailer.createTransport({
```

Comment:
Add error handling for transporter.sendMail to capture cases where the email server might be down.